docs/operations-guide/running-metabase-on-elastic-beanstalk.md

```diff
@@ -1,283 +1,258 @@
+- For `Instance class` you can choose any size, but we recommend `db.t2.small` or larger for production installs. Metabase is pretty efficient so there is no need to make this a big instance.
+- You can safely leave `Storage` to the default size.
+- Pick a `Username` and `Password` for your database. We suggest you hold onto these credentials in a password manager, as it can be useful for things like backups or troubleshooting. These settings will be automatically made available to your Metabase instance, so you will not need to put them in anywhere manually.
+- You can safely leave the `Retention setting` as `Create snapshot`.
+- Under `Availability` we recommend the default value of `Low (one AZ)` for most circumstances.
 
+![Elastic Beanstalk Database Settings](images/EBDatabaseSettings.png)
 
+Once you've entered a password and clicked `Save`, the red outline on the Database box should have gone away, indicating that the application is valid and ready to be launched.
 
 ### Additional Options
 
+#### Using Metabase in a VPC
 
-##### Using Metabase in a VPC
+Newer AWS accounts are encouraging the use of Virtual Private Cloud (VPC) for deployments, and in general we think it's simplest to follow that best practice.
 
-Newer AWS accounts are encouraging the use of VPC for deployments and in general we think it's simplest to follow that best practice.
+If you prefer not to use a VPC, that's fine. But one thing to note is that some EC2 instance types (`t2.*` specifically) are not available outside of a VPC, so if you choose to not use a VPC then make sure and pick appropriate instance types.
 
-If you prefer not to use a VPC that is fine, however one thing to note is that some EC2 instance types (t2.* in specific) are not available outside of a VPC, so if you choose to not use a VPC then make sure and pick appropriate instance types.
+If you are launching your Metabase inside of a VPC you'll now need to check a few boxes to enable your application to work inside your VPC subnets. Click `Modify` on this Network box to set things up.
 
 ![Elastic Beanstalk VPC Entry](images/EBVPCEntry.png)
 
-If you are launching your Metabase inside of a VPC you'll now need to check a few boxes to enable your application to work inside your VPC subnets.
-
-Unless you have a custom VPC setup that you know how to configure it's easiest to just check all the boxes and allow your infrastructure to exist on all subnets. Note that the Load Balancer cannot be in more than one subnet per availability zone, and the database requires subnets in at least two availability zones.
+Unless you have a custom VPC setup that you know how to configure, it's easiest to just check all the boxes and allow your infrastructure to exist on all subnets. Note that the Load Balancer cannot be in more than one subnet per availability zone, and the database requires subnets in at least two availability zones. Once you've finished your VPC config click `Save`.
```

Comment:
FYI: this is what I see when I click "Modify" in the "Network" block (sans the blur, of course :wink:): ![EB - Create Application - Modify network page](https://user-images.githubusercontent.com/23798/66216444-c8018780-e6c5-11e9-99f3-b3a7845ecc4f.png)

I believe those are the checkboxes that should all be checked.

The full value for **VPC**, selected by default, is `vpc-12345678 (<ip range>) | VPC for everything (default)`.

I am unsure which of these default values may be influenced by our specific setup.

Before clicking "Modify" on the "Network" block (or cancelling out of the above screen), it looks like this: 
![image](https://user-images.githubusercontent.com/23798/66216919-a3f27600-e6c6-11e9-816f-fef4c3516890.png)

I'm not sure if anything really needs updating, but I just wanted to clarify what this is describing.